drivers/windows/windows.go

```diff
@@ -431,6 +431,7 @@ func convertQosPolicies(qosPolicies []types.QosPolicy) ([]json.RawMessage, error
 	for _, elem := range qosPolicies {
 		encodedPolicy, err := json.Marshal(hcsshim.QosPolicy{
 			Type: "QOS",
+
```

Comment:
This is a (dirty) workaround for `gofmt` changes between Go 1.10 and Go 1.11 and up. Adding this blank line makes it produce the same formatting on both versions (thus preventing CI linting failures if it checks for `gofmt`)